index.js

```diff
@@ -39,6 +40,19 @@ exports.deploy = async function (options) {
     const account = await near.account(options.accountId);
     let prevState = await account.state();
     let prevCodeHash = prevState.code_hash;
+
```

Comment:
I would move `Starting deployment...` console.log to the place where it's actually happening now.